### PR TITLE
remove test variants, update commit id

### PR DESCRIPTION
**Pull request by vanhauser-thc**
this leaves one changed temporary variant.
the optimal variant has larger changes which might result in build failures, so I will watch this and update accordingly if necessary.

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
uh all checks passed! I am amazing! :)

File: fuzzers/aflplusplus_qemu/builder.Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG parent_image
+ARG parent_image=gcr.io/fuzzbench/base-builder
```
Comment (inferno-chromium) [inline fuzzers/aflplusplus_qemu/builder.Dockerfile:15]: Can you remove the gcr.io argument in all the files for parent_image, it is not needed after https://github.com/google/fuzzbench/commit/eee4f8d22449d4e6c4345a57c0e81958da4287c4

File: fuzzers/aflplusplus_optimal/fuzzer.py [hunk 2]
```diff
@@ -31,35 +31,37 @@ def build():  # pylint: disable=too-many-branches,too-many-statements
     """Build benchmark."""
     benchmark_name = os.environ['BENCHMARK']
 
-    if benchmark_name == 'bloaty_fuzz_target':
-        aflplusplus_fuzzer.build("lto", "laf", "autodict", "dynamic")
-    elif benchmark_name == 'curl_curl_fuzzer_http':
+    if benchmark_name == 'curl_curl_fuzzer_http':
```
Comment (inferno-chromium) [inline fuzzers/aflplusplus_optimal/fuzzer.py:34]: Do you want anything for bloaty, that benchmark is still there.

File: .github/workflows/fuzzers.yml [hunk 3]
```diff
@@ -33,13 +33,7 @@ jobs:
           - aflplusplus_qemu
           - honggfuzz_qemu
           # temporary variants.
```
Comment (inferno-chromium) [inline .github/workflows/fuzzers.yml:35]: Is aflplusplus_havoc a temporary variant, if not this comment can be removed.

File: fuzzers/aflplusplus_qemu/builder.Dockerfile (hunk 1 above)
Comment (vanhauser-thc) [reply fuzzers/aflplusplus_qemu/builder.Dockerfile:15]: done!

File: fuzzers/aflplusplus_optimal/fuzzer.py (hunk 2 above)
Comment (vanhauser-thc) [reply fuzzers/aflplusplus_optimal/fuzzer.py:34]: bloaty is using the default now (LTO with a dynamic map and cmplog/redqueen)

File: .github/workflows/fuzzers.yml (hunk 3 above)
Comment (vanhauser-thc) [reply .github/workflows/fuzzers.yml:35]: yes that is a temporary variant (changed the havoc iteration algo, the previous attempt was not good, hopefully this one performs better)

**Pull request merged**